packages/app-store/hubspot/lib/CrmService.ts

```diff
@@ -234,49 +178,11 @@ export default class HubspotCalendarService implements Calendar {
     return Promise.reject("Something went wrong when creating a meeting in HubSpot");
   }
 
-  async createEvent(event: CalendarEvent): Promise<NewCalendarEventType> {
```

Comment:
In every CRM app when creating events, there was logic to determine if contacts had to be created. I moved this into the `CrmManager`.